app/controllers/claims/support/claims_controller.rb

```diff
@@ -1,11 +1,2 @@
 class Claims::Support::ClaimsController < Claims::Support::ApplicationController
-  before_action :set_school, only: [:index]
```

Comment:
If there's nothing in this controller is it necessary?

Reply:
The controller is used to map to the default Rails `index` action. It only renders the view template which has no variables, hence no setup in the controller.

![CleanShot 2024-01-15 at 09 22 51](https://github.com/DFE-Digital/itt-mentor-services/assets/4231530/ba2439e0-6610-41c9-8df6-031b9cd21bc8)